service/index/reader.go

```diff
@@ -47,29 +48,29 @@ func NewReader(db *badger.DB) *Reader {
 // First returns the height of the first finalized block that was indexed.
 func (r *Reader) First() (uint64, error) {
 	var height uint64
-	err := r.db.View(storage.RetrieveFirst(&height))
+	err := r.db.View(r.storage.RetrieveFirst(&height))
```

Comment:
I like `lib` better because it's shorter, but both work for me as long as it is consistent.